client/packages/programs/src/JsonForms/common/components/Array/Notes.tsx

```diff
@@ -191,25 +195,24 @@ const NoteComponent = (props: ControlProps) => {
     // TO-DO: Use full name for Author name once available in database
     const authorName = config.user?.name;
     const created = data?.created ?? new Date().toISOString();
-    handleChange(
-      path,
-      error
-        ? undefined
-        : {
-            text,
-            authorId,
-            authorName,
-            created,
-          }
-    );
+    if (text)
+      handleChange(
+        path,
+        error
+          ? undefined
+          : {
+              text,
+              authorId,
+              authorName,
+              created,
+            }
+      );
```

Comment:
Only update with correct data if text is valid. There is a separate call to "handleChange" in the input `onBlur` property, which puts some "invalid" data into the form data.

Reply:
Nit: We don't do it elsewhere but something like more subtle "required asterisk *" might be nicer because it might still be confusing to have the save button disabled without visible error hint...